renv/settings.json

```diff
@@ -9,7 +9,7 @@
     "Depends",
     "LinkingTo"
   ],
-  "ppm.enabled": null,
+  "ppm.enabled": true,
```

Comment:
This just gives preference to using binary packages from PPM instead of source tarballs.